apps/remote_control/test/lexical/remote_control/search/indexer/extractors/function_definition_test.exs

```diff
@@ -165,6 +165,54 @@ defmodule Lexical.RemoteControl.Search.Indexer.Extractors.FunctionDefinitionTest
       assert decorate(doc, function.block_range) =~ "«def do_proto(a, b) do\n    a + b\n  end»"
     end
 
+    test "finds functions defined with defdelegate" do
+      {:ok, [function | _], doc} =
+        ~q[
+          defmodule MyModule do
+            defdelegate map(enumerable, other), to: Enum
+          end
+        ]
+        |> index_functions()
+
+      assert function.type == {:function, :delegate}
+      assert function.subject == "MyModule.map/2"
+      assert "map(enumerable, other)" = extract(doc, function.range)
+      assert decorate(doc, function.range) =~ "defdelegate «map(enumerable, other)», to: Enum"
+    end
+
+    test "finds functions defined with defdelegate with an aliased module" do
+      {:ok, [function | _], doc} =
+        ~q[
+          defmodule MyModule do
+            alias Parent.Child
+            defdelegate map(enumerable, other), to: Child
```

Comment:
My only wish for the delegate is that when the cursor is on `Mymodule.map|`, I hope it can quickly go to the definition of `Child.map` as well. So, in this case, should we return two definitions? With the existing LS, trying to jump to the original function's definition is really difficult, or even impossible to jump to.

Reply:
We can definitely return two definitions for delegate functions.

Reply:
oh heh, the reason it does this is because we're still using elixirsense to find definitions for functions.

Reply:
...and that should be a separate PR.

Reply:
> When I find a function's definition, I might need to determine whether the definition's location is that of a delegate definition and its delegate reference, and then look for the original function's definition?

No, when you find a function's definition entry, you check the type of it, and if it's `{:function, :delegate}` then you know it's a delegated function and can look up the original. Another option would be to emit another definition entry for the function that's the target of `defdelegate`, then it'd just be two results for the definition call, but I don't know if that's semantically correct.